<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>751</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* ===== 外观：柔和渐变 + 圆角 + 细描边 ===== */
QDialog#LoginDialog {
    background: qlineargradient(x1:0,y1:0, x2:0,y2:1,
                                stop:0   #eef5ff,
                                stop:0.55 #f3f7ff,
                                stop:1   #fbfcff);
    border-radius: 20px;
    border: 1px solid rgba(0,0,0,0.06);
}
QWidget#title_widget, QWidget#center_widget {
    background: transparent;
    border-radius: 20px;
}
QWidget#title_widget {
    background: rgba(255,255,255,0.26);
    border-top-left-radius: 20px;
    border-top-right-radius: 20px;
}

/* 标题 */
QLabel#titleLabel {
    color: #111827;
    font-size: 18px;               /* 更有分量 */
    font-weight: 600;
    letter-spacing: .2px;
}

/* 顶部两个圆形工具按钮：更柔和的悬停/按下态 */
QToolButton#toolButton_github,
QToolButton#toolButton_close {
    background: transparent;
    border: none;
    width: 20px; height: 20px;
    border-radius: 14px;
    qproperty-iconSize: 18px 18px;
}

QToolButton#toolButton_close      { border-image: url(:/image/x-circle.svg); }
QToolButton#toolButton_github    { border-image: url(:/image/github.svg); }

QToolButton#toolButton_close:hover {
    border-image: url(:/image/x-circle-fill.svg); /* 悬停时用另一张图 */
}
QToolButton#toolButton_close:pressed{ background: rgba(0,0,0,0.12); }

/* 大图标容器（label_5）做成浅卡片 */
QLabel#label_5 {
    background: rgba(255,255,255,0.75);
    border: 1px solid rgba(0,0,0,0.05);
    border-radius: 18px;
}

/* ===== 输入框 / 下拉框：更“胖”的圆角卡片 ===== */
QLineEdit, QComboBox {
    background: #ffffff;
    border: 1px solid rgba(0,0,0,0.12);
    border-radius: 16px;           /* 圆角更大 */
    padding: 10px 14px;            /* 内边距更足 -&gt; 更“胖” */
    min-height: 42px;              /* 统一高度 */
    font-size: 14px;
    color: #111827;
    selection-background-color: #5CAEFF;
    selection-color: #ffffff;
}
QLineEdit:focus, QComboBox:focus {
    border: 1px solid #5CAEFF;
    outline: 4px solid rgba(92,174,255,0.18);   /* 柔和的焦点光晕 */
}
QComboBox::drop-down { width: 26px; border: none; background: transparent; margin-right: 6px; }
QComboBox QAbstractItemView {
    background: #ffffff;
    border: 1px solid rgba(0,0,0,0.12);
    selection-background-color: #E8F2FF;
    outline: none;
}


/* 下拉列表美化 */
QComboBox#userTypeCombo::drop-down {
    width: 26px; border: none; background: transparent; margin-right: 6px;
}
QComboBox#userTypeCombo QAbstractItemView {
    background: #ffffff;
    border: 1px solid rgba(0,0,0,0.12);
    outline: none;
    selection-background-color: #E8F2FF;
}

/* 单选框（你的 .ui 用的是 QRadioButton） */
QRadioButton#radioButton {
    color: #6b7280;  /* 次要文本色 */
    spacing: 8px;
}
QRadioButton#radioButton::indicator {
    width: 16px; height: 16px;
    border-radius: 4px;
    border: 1px solid #BFC6D1;
    background: #fff;
}
QRadioButton#radioButton::indicator:checked {
    background: #5CAEFF;
    border: 1px solid #5CAEFF;
}

/* ===== 主按钮：更“胖”的胶囊纽（圆角更大） ===== */
QPushButton#loginButton {
    background: qlineargradient(x1:0,y1:0, x2:0,y2:1,
                                stop:0 #6fb2ff, stop:1 #4ea4ff);
    border: none;
    border-radius: 22px;       /* 胶囊 */
    color: #ffffff;
    min-height: 54px;          /* 更胖 */
    font-size: 15px;
    font-weight: 600;
    letter-spacing: .2px;
}
QPushButton#loginButton:hover  {
    background: qlineargradient(x1:0,y1:0, x2:0,y2:1,
                                stop:0 #84bfff, stop:1 #61afff);
}
QPushButton#loginButton:pressed { background: #3e93ea; }

/* 次级按钮（“快速注册”“获取帮助”）：文字按钮（无背景/无边框） */
QPushButton:not(#loginButton) {
    background-color: rgba(0, 0, 0, 0);
    border: none;
    color: #2D7EF7;
    font-size: 13px;
    min-height: 28px;
    padding: 4px 6px;
}
QPushButton:not(#loginButton):hover   { color: #1B63D9; text-decoration: underline; }
QPushButton:not(#loginButton):pressed { color: #114DB3; }

/* 更轻的分隔线与版权 */
Line { color: rgba(0,0,0,0.10); }
QLabel#label_edit { 
	color: rgba(195, 195, 195,255); 
    font-size: 12px;
}

/* 禁用态 */
*:disabled { color: rgba(17,24,39,0.34); }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="title_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>461</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_github">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../image/UiTool.qrc">
          <normaloff>:/login/github.svg</normaloff>:/login/github.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_close">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../image/UiTool.qrc">
          <normaloff>:/login/x-circle.svg</normaloff>:/login/x-circle.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>工业现场远程专家支持系统</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="center_widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_profile">
          <property name="minimumSize">
           <size>
            <width>95</width>
            <height>90</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="margin">
           <number>15</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="userTypeCombo">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>64</height>
           </size>
          </property>
          <property name="currentText">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="usernameEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>64</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>请输入用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>64</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>请输入密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="LogAgreeButton">
        <property name="text">
         <string>已阅读并同意服务协议和隐私保护指引</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loginButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>54</height>
         </size>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="registerButton">
          <property name="text">
           <string>快速注册</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="helpButton">
          <property name="text">
           <string>获取帮助</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_edit">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>© 2025 工业现场远程专家支持系统 v1.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../image/UiTool.qrc"/>
 </resources>
 <connections/>
</ui>
